study-builder/studies/testboston/study-events.conf

```diff
@@ -240,7 +277,7 @@
           || user.studies["testboston"].forms["BASELINE_COVID"].questions["INTERESTED_CONTINUING"].answers.hasFalse()
           """,
       "maxOccurrencesPerUser": 1,
-      "delaySeconds" : 21600, #6 hrs
+      "delaySeconds": 21600, # 6 hrs
```

Comment:
Adhoc symptom survey instance creation cannot be driven by any DSM event and still need to be time driven !!!

Reply:
Isn't the specific requirement for adhoc symptom survey is to be created 6 hours after, and not triggered by any other thing? So I think that's fine.

Reply:
Just like baseline symptom survey I assumed at the time of requirement no dsm event to trigger this adhoc survey and hence only option was time bound. Now that we are chaning baseline just wondered if there is a way to trigger by any of new DSM events. looks like not, thats fine.